build.gradle

```diff
@@ -382,6 +433,13 @@ dependencies {
     testCompileOnly group: 'org.projectlombok', name: 'lombok'
     testAnnotationProcessor group: 'org.projectlombok', name: 'lombok'
 
+    //pact contract testing
+    contractTestImplementation group: 'au.com.dius.pact.consumer', name: 'junit5', version: '4.4.0-beta.2'
+    contractTestImplementation group: 'org.springframework.boot', name: 'spring-boot-starter-test', version: '2.7.12'
```

Comment:
are you sure we need spring boot version 2.7? couple months ago we have migrated to spring boot v3.

Reply:
Removed this line as we dont need it